src/symon/index.ts

```diff
@@ -286,6 +259,40 @@ export default class SymonClient {
     this.probeChangesCheckedAt = probeChangesCheckedAt
   }
 
+  private async fetchAndApplyProbeChanges() {
```

Comment:
it works on my machine 🦾 🤖 

btw i think i found an edge case. I don't know if this is covered in this task.

How to reproduce:
1. run symon, create monitor with location A.
2. run monika using your described config with loaction B
3. change the location of created monitor in symon to location B
4. the config is not detecting change

Reply:
It needs to be fixed in the Symon. I will try to fix it later.